js/components/attorney-banner/attorney-banner.js

```diff
@@ -0,0 +1,51 @@
+// This code is based on original code from assets-frontend: https://github.com/hmrc/assets-frontend/blob/main/assets/javascripts/modules/attorneyBanner.js
+import getWindowTop from './get-window-top';
+import getElementRectangle from './get-element-rectangle';
+
+const bannerPlaceholderClass = 'pta-attorney-banner__placeholder';
+const bannerFixedModifierClass = 'pta-attorney-banner--fixed';
+
+function insertPlaceholder(module, originalRectangle) {
+  const placeholder = document.createElement('div');
+  placeholder.className = bannerPlaceholderClass;
+  placeholder.style.width = `${originalRectangle.width}px`;
+  placeholder.style.height = `${originalRectangle.height}px`;
+  placeholder.style.display = 'none';
+  module.parentNode.insertBefore(placeholder, module);
+  return placeholder;
+}
+
+function fixBanner(module, placeholder) {
+  module.classList.add(bannerFixedModifierClass);
+  // eslint-disable-next-line no-param-reassign
+  placeholder.style.display = 'block';
+}
+
+function unfixBanner(module, placeholder) {
+  module.classList.remove(bannerFixedModifierClass);
+  // eslint-disable-next-line no-param-reassign
+  placeholder.style.display = 'none';
+}
+
+function AttorneyBanner($module) {
+  this.$module = $module;
```

Comment:
@matthewmascord no worries if this is copying another convention but the naming for this confused me, I didn't know what it meant by $module, what do you think about $el or $element ?

Reply:
this is just following the convention used by govuk-frontend/hmrc-frontend